docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -174,6 +174,66 @@ local_assets = with_resources(
 )
 ```
 
+### Customizing how assets are loaded downstream
```

Comment:
i feel pretty uncertain about where this is placed in the page. Thematically it fits with the asset IO section, but it relies on the user know how IO Managers are defined, which isn't covered until the next section. I *could* shorten this to be more like the per-output section, and then show the full code snippet in an example, but i'd be worried that some important context (that input managers need to have some knowledge of the how the output was stored) would be missed by users who don't look at the example. what do you think?

Reply:
Would it possibly make sense to introduce setting input managers before introducing how to define them? Similar to what we do with IO managers?

Reply:
rearranged the docs section, lmk how it reads now